src/main/groovy/org/transmartproject/batch/concept/postgresql/PostgresInsertConceptCountsTasklet.groovy

```diff
@@ -21,19 +22,26 @@ class PostgresInsertConceptCountsTasklet extends InsertConceptCountsTasklet {
             AND sourcesystem_cd = ?
         ),
         code_patients AS (
-          SELECT concept_path, concept_cd, patient_num
+          SELECT
+            concept_path::text,
+            substring(concept_path from '#"%\\#"%\\' for '#') as parent_concept_path,
+            patient_num
           FROM
             i2b2demodata.observation_fact
             NATURAL INNER JOIN relevant_concepts
+          UNION
+          SELECT
+            parent_concept_path as concept_path,
+            substring(parent_concept_path from '#"%\\#"%\\' for '#') AS parent_concept_path,
+            patient_num
+          FROM code_patients WHERE parent_concept_path != ?
```

Comment:
From what I understand the placeholder is bound to the to the parent of the base path being updated and in the WHERE condition `parent_concept_path` refers to the second column in the current set of rows (as opposed to those from the previous iteration). Doesn't this stop a node too soon? Shouldn't nodes should be excluded only when the `concept_path` (rather than the `parent_concept_path`) refers to the the parent of the base?